### ISSUE TITLE
the pipe character | within the \w and \zalns has a space after it on export

### ISSUE THREAD

**Issue by birchamp** [unfoldingWord/translationCore#7165]
A space should not be inserted after the pipe character, but on export these are being inserted.

**Comment by birchamp** [comment]
Please check with @RobH123 for examples of how it should be vs how it is currently exported.

**Comment by da1nerd** [comment]
@RobH123 is this with usfm 2 or 3? 

Edit: after looking at the code, I don't think it matters.

**Comment by da1nerd** [comment]
@RobH123 here's some sample text. Is this what you want (in the green), or do I need to remove the space in front of the pipe as well?

```diff
\id MAT
\c 4
\v 6
+ \zaln-s |x-content="καὶ" x-lemma="καί" x-morph="Gr,CC,,,,,,,," x-occurrence="1" x-occurrences="2" x-strong="G25320"
- \zaln-s | x-content="καὶ" x-lemma="καί" x-morph="Gr,CC,,,,,,,," x-occurrence="1" x-occurrences="2" x-strong="G25320"
\w and|x-occurrence="1" x-occurrences="1"\w*
\zaln-e\*
```

**Comment by RobH123** [comment]
You're correct @da1nerd : Just remove the space AFTER the pipe. I'm just looking at examples on http://ubsicap.github.io/usfm/master/attributes/index.html and http://ubsicap.github.io/usfm/master/milestones/index.html.

USFM2 is obsolete for new projects, so we shouldn't ever be writing it I don't believe.

**Comment by PhotoNomad0** [comment]
@elsylambert verified in translationCore 3.0.2 (c8c2941)

**Comment by elsylambert** [comment]
Verified in translationCore 3.0.2 (c8c2941) and found to be working as expected. No white space after pipe character | and x  adds up after Export to USFM.
[43-LUK-test.usfm.zip](https://github.com/unfoldingWord/translationCore/files/7570781/43-LUK-test.usfm.zip)

**Comment by elsylambert** [comment]
Looks good in translationCore 3.0.2 (0d9cd25).

**Comment by gayathrihayden** [comment]
Fix tested in tC 3.0.2 (0d9cd25)  and works fine. 

Reference screenshots:

In tC 3.0.1 (343198b), observe space after the pipe character |:

![tC_3_0_1-space after pipe character](https://user-images.githubusercontent.com/74929193/143138016-080a776a-df90-4d95-a44a-1e16ed407ca0.PNG)

In tC 3.0.2 (0d9cd25), do not observe space after the pipe character |:

![tC_3_0_2-no space after pipe character](https://user-images.githubusercontent.com/74929193/143138095-e5d49d18-5d96-4a43-a81f-efc8aef96efc.PNG)
